yadm

```diff
@@ -827,7 +827,7 @@ function clone() {
   rm -rf "$wc"
 
   # then reset the index as the --no-checkout flag makes the index empty
-  "$GIT_PROGRAM" reset --quiet -- .
+  "$GIT_PROGRAM" reset --quiet -- "$YADM_WORK"
```

Comment:
Consider using `:`/ instead of `$YADM_WORK` to match the pathspec used below (in the checkout command).

Reply:
Nice info, I just know about pathspec
Changed in ae3a1494497702d16f2331a8d39c136c6e11f066